creme/creme_core/views/generic/order.py

```diff
@@ -56,6 +58,7 @@ class ReorderInstances(View):
     target_order_post_argument: str = 'target'
     order_field_name: str = 'order'
     use_select_for_update = True
+    use_raw_update = True
```

Comment:
Hum set to False by default to keep the current behaviour on existing views.
Save() is cleaner (update() bypasses the signal etc...) so it's better as default behaviour.

Reply:
Should I keep raw update for the 'creme_config__reorder_menu_level0' view ?